### ISSUE TITLE
Templatetag js_reverse_inline breaks on Django 1.9

### ISSUE THREAD

**Issue by tommikaikkonen** [ierror/django-js-reverse#42]
I'm using the `{% js_reverse_inline %}` template tag to put the script on a couple of pages in a project. Those pages started failing with a JavaScript SyntaxError once the project was updated from Django 1.8 to 1.9, which looks to be because of [this change in template tag escaping](https://docs.djangoproject.com/en/1.9/releases/1.9/#simple-tag-now-wraps-tag-output-in-conditional-escape). The output of the `simple_tag` is escaped by default now and a SyntaxError is raised when the parser hits `&amp;&amp;`.

You can work around the error by turning autoescape off around the tag:

```html
{% autoescape off %}
<script type="text/javascript" charset="utf-8">{% js_reverse_inline %}</script>
{% endautoescape %}
```

Looking at the release notes, the proper fix seems to be to wrap the return value of `js_reverse_inline` with `mark_safe`.

**Comment by ierror** [comment]
Thank you for reporting! I'll take a look the next days. Note to self: Write better tests... https://github.com/ierror/django-js-reverse/blob/develop/tests/unit_tests.py#L265

**Comment by dtran320** [comment]
@ierror I tried to add a failing test case for `py-django1.9` for this in `JSReverseTemplateTagTest`, but we'd need to catch a JS error after the JS injected by `{% js_reverse_inline %}` gets run. 

There are a couple of approaches that might work-- looking into the `selenium.webdriver.phantomjs.webdriver.WebDriver`'s log, or adding an `onerror` event that adds something to the DOM whose presence we can check for.

**Comment by ierror** [comment]
Fixed and released v0.7.2 to pypi - Thanks for your debugging @tommikaikkonen

**Comment by ierror** [comment]
@dtran320 my solution is to compare the template tag-output-js to the one of the view-js. https://github.com/ierror/django-js-reverse/blob/develop/tests/unit_tests.py#L268 - the view-js is checked by selenium on other tests, so i think it's valid...
